_posts/2015-09-29-Asking-Users-for-Help.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: Asking users for help
+permalink: /asking-users-for-help/
+tags: android, debugging
+category: Android
+author_id: mbobzien
+---
+
+##What’s this article about?
```

Comment:
Maybe you can skip the first title, our current theme picks it as preview for the article when listing it on the front page instead of the following paragraph 

![frontpage](https://cloud.githubusercontent.com/assets/684621/10841099/b7ba2ecc-7ee6-11e5-9ba2-f655fca665ad.png)